pkgs/tools/networking/redfang/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitLab, bluez }:
+
+stdenv.mkDerivation rec {
+  pname = "redfang";
+  version = "2.5";
+
+  src = fetchFromGitLab {
+    group = "kalilinux";
+    owner = "packages";
+    repo = pname;
+    rev = "upstream/${version}";
+    sha256 = "sha256-dF9QmBckyHAZ+JbLr0jTmp0eMu947unJqjrTMsJAfIE=";
+  };
+
+  patches = [ ./makefile-install.diff ];
+
+  installFlags = [ "DESTDIR=$(out)" ];
+
+  NIX_CFLAGS_COMPILE = "-Wno-format-security";
+
+  buildInputs = [ bluez ];
+
+  meta = with lib; {
+    description = "A small proof-of-concept application to find non discoverable bluetooth devices";
+    homepage = "https://gitlab.com/kalilinux/packages/redfang";
+    license = licenses.unfree; # Upstream has no license
```

Comment:
License seems to be [`GPL2`](https://gitlab.com/kalilinux/packages/redfang/-/blob/kali/master/fang.c#L7-9).